protos/ordererpb/ordererpb.proto

```diff
@@ -11,7 +11,9 @@ import "mir/codegen_extensions.proto";
 import "net/codegen_extensions.proto";
 
 message Event {
```

Comment:
Alignment in consistent with the rest of the protobuf definitions.

Reply:
I could not be certain of what you meant here, is it fixed now?

Reply:
Actually, sorry, I made a mistake which lead to nitpicking. But now it's beautiful :-)